alacritty_terminal/src/event_loop.rs

```diff
@@ -265,6 +268,10 @@ where
                                     // With hold enabled, make sure the PTY is drained.
                                     let _ = self.pty_read(&mut state, &mut buf, pipe.as_mut());
                                 } else {
+                                    // We should be able to drain the PTY without hold.
+                                    if self.drain_pty {
+                                        let _ = self.pty_read(&mut state, &mut buf, pipe.as_mut());
+                                    }
```

Comment:
I think it would be wise to combine these two statements, just in case of future changes.

So just a `if self.hold || self.drain_pty`, we can then just do `if !self.hold` for shutdown; the double check doesn't really have any negative impact here.

Reply:
This is much simpler now after removing `hold`.